src/NServiceBus.Core.Analyzer.Tests/AwaitOrCaptureTasksAnalyzerTests.cs.bak

```diff
@@ -0,0 +1,273 @@
+namespace NServiceBus.Core.Analyzer.Tests
```

Comment:
was this .cs.bak file meant to make it to the PR?

Reply:
Definitely not! Removed.